2023-10-15 22:47:30,355 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Configure stats pid to 90933
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Loading settings from /Users/arygup/.config/wandb/settings
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Loading settings from /Users/arygup/Desktop/A3/wandb/settings
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_init.py:_log_setup():528] Logging user logs to /Users/arygup/Desktop/A3/wandb/run-20231015_224730-ad165v09/logs/debug.log
2023-10-15 22:47:30,356 INFO    MainThread:90933 [wandb_init.py:_log_setup():529] Logging internal logs to /Users/arygup/Desktop/A3/wandb/run-20231015_224730-ad165v09/logs/debug-internal.log
2023-10-15 22:47:30,357 INFO    MainThread:90933 [wandb_init.py:init():568] calling init triggers
2023-10-15 22:47:30,357 INFO    MainThread:90933 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'architecture': 'Logistic Regression', 'dataset': 'wineQT'}
2023-10-15 22:47:30,357 INFO    MainThread:90933 [wandb_init.py:init():593] re-initializing run, found existing run on stack: 2udn7fpb
2023-10-15 22:47:30,359 INFO    MainThread:90933 [wandb_run.py:_finish():1934] finishing run smaia3/Q1/2udn7fpb
2023-10-15 22:47:30,359 INFO    MainThread:90933 [jupyter.py:save_history():445] not saving jupyter history
2023-10-15 22:47:30,360 INFO    MainThread:90933 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-10-15 22:47:30,360 INFO    MainThread:90933 [wandb_init.py:_jupyter_teardown():456] cleaning up jupyter logic
2023-10-15 22:47:30,360 INFO    MainThread:90933 [wandb_run.py:_atexit_cleanup():2168] got exitcode: 0
2023-10-15 22:47:30,360 INFO    MainThread:90933 [wandb_run.py:_restore():2151] restore
2023-10-15 22:47:30,361 INFO    MainThread:90933 [wandb_run.py:_restore():2157] restore done
2023-10-15 22:47:43,427 INFO    MainThread:90933 [wandb_init.py:init():618] starting backend
2023-10-15 22:47:43,427 INFO    MainThread:90933 [wandb_init.py:init():622] setting up manager
2023-10-15 22:47:43,428 INFO    MainThread:90933 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-10-15 22:47:43,429 INFO    MainThread:90933 [wandb_init.py:init():628] backend started and connected
2023-10-15 22:47:43,437 INFO    MainThread:90933 [wandb_run.py:_label_probe_notebook():1276] probe notebook
2023-10-15 22:47:43,437 INFO    MainThread:90933 [wandb_run.py:_label_probe_notebook():1286] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-10-15 22:47:43,437 INFO    MainThread:90933 [wandb_init.py:init():720] updated telemetry
2023-10-15 22:47:43,437 INFO    MainThread:90933 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
